src/libgeodecomp/loadbalancer/loadbalancer.h

```diff
@@ -4,14 +4,20 @@
 #include <libgeodecomp/misc/stdcontaineroverloads.h>
 #include <vector>
 
+#if defined(LIBGEODECOMP_WITH_HPX)
+#include<hpx/config.hpp>
+#elif !defined(HPX_COMPONENT_EXPORT)
+#define HPX_COMPONENT_EXPORT
```

Comment:
Can I be sure that the definition of this macro won't confuse other HPX headers that might get included afterwards? My naive understanding is that it would be safer to use the conditional below when the class LoadBalancer is defined.